tasks.py

```diff
@@ -59,7 +59,7 @@ def build(c, output='output', ziparchive=None):
     args = [
         'pip', 'install',
         '-r', tmp.name,
-        '--python-version', '37',
+        #'--python-version', '37',
```

Comment:
one of our installers uses a deprecated function that was removed in newer versions of python. build will error if python 3.7 is not used.

Reply:
reverted